kura/maven-central/p2tom2.properties

```diff
@@ -0,0 +1,76 @@
+# mirror.<iu>=false
+# dependency.<ns>.<name>.ignore = true|false
+# mapping.<iu>=groupId:artifactId[:version]
+# external.<iu>=groupId:artifactId[:version]
+
+# Select default mapping strategy
+
+implementation.mavenMapping=org.eclipse.scada.releng.p2.to.maven.DefaultMavenMapping
+scrubJars=true
+
+# Define project metadata
+
+default.project.url=http://eclipse.org/kura
+
+developers=ctron
+
+developer.ctron.name=Jens Reimann
+developer.ctron.email=jreimann@redhat.com
+developer.ctron.url=https://dentrassi.de
+developer.ctron.organization=Red Hat Inc
+developer.ctron.organizationUrl=https://redhat.com
+
```

Comment:
As discussed today we will go with the simple approach of listing the contributors page like here [1]

[1] https://search.maven.org/#artifactdetails|org.eclipse.platform|org.eclipse.swt|3.105.2|jar